core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -638,7 +638,7 @@ object DumpLogSegments {
       " Instead, the value-decoder-class option can be used if a custom RLMM implementation is configured.")
     private val shareStateOpt = parser.accepts("share-group-state-decoder", "If set, log data will be parsed as share group state data from the " +
       "__share_group_state topic.")
-    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing metadata for each record.")
+    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing metadata for each record and control records.")
```

Comment:
```suggestion
    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing metadata for control and all other records.")
```
each should include control as well.

Maybe put the exception or important first things first.

Reply:
We skip the entire control record, both data and metadata, but I rephrased similar to what you suggest.

Reply:
thanks, sounds better now.